content/influxdb/v2.0/reference/internals/shards.md

```diff
@@ -0,0 +1,159 @@
+  - [Shard group duration](#shard-group-duration)
+  - [Shard group diagram](#shard-group-diagram)
+- [Shard life-cycle](#shard-life-cycle)
+  - [Shard precreation](#shard-precreation)
+  - [Shard states](#shard-states)
+  - [Shard compaction](#shard-compaction)
+- [Shard deletion](#shard-deletion)
+
+## Shards
+A shard contains encoded and compressed time series data for a given time range
+defined by the [shard group duration](#shard-group-duration).
+All points in a [series](#series) within the shard's time range are stored in the same shard.
+A single shard contains multiple series, multiple [TSM files](#tsm-time-structured-merge-tree) on disk,
+and belongs to a [shard group](#shard-groups).
+
+## Shard groups
+Shard groups are logical containers for InfluxDB shards.
+A single shard group contains time series data for a specific time range defined by
+the [shard group duration](#shard-group-duration).
+A shard group belongs to an InfluxDB [bucket](/influxdb/v2.0/reference/glossary/#bucket).
+
+{{% note %}}
+In **InfluxDB OSS**, a shard group typically contains only a single shard.
+In an **InfluxDB Enterprise 1.x cluster**, shard groups contain multiple shards
+distributed across multiple data nodes.
+{{% /note %}}
+
+### Shard group duration
+The **shard group duration** determines the time range a shard group covers
+and the interval at which new shard groups are created.
+By default, InfluxDB determines shard group durations using
+the [retention period](/influxdb/v2.0/reference/glossary/#retention-period)
+of the bucket:
+
+| Bucket retention period     | Default shard group duration |
+|:-----------------------     | ----------------------------:|
+| less than 2 days            | 1h                           |
+| between 2 days and 6 months | 1d                           |
+| greater than 6 months       | 7d                           |
+
+#### Manage shard group durations
+To configure a custom bucket shard group duration, use the `--shard-group-duration`
+flag with the [`influx bucket create`](/influxdb/v2.0/reference/cli/influx/bucket/create/#create-a-custom-shard-group-duration)
+and [`influx bucket update`](/influxdb/v2.0/reference/cli/influx/bucket/update//#update-the-shard-group-duration-of-a-bucket) commands.
+
+{{% note %}}
+Shard group durations must be shorter than the bucket's retention period.
+{{% /note %}}
+
+To view your bucket's shard group duration, use the
+[`influx bucket list` command](/influxdb/v2.0/reference/cli/influx/bucket/list/).
+
+### Shard group diagram
+The following diagram represents a **bucket** with a **4d retention period**
+and a **1d shard group duration**:
+
+---
+
+{{< html-diagram/shards >}}
+
+---
+
+## Shard life-cycle
+
+### Shard precreation
+The InfluxDB **shard precreation service** creates new shards before data actually arrives.
+The precreator service only creates shards with future start and end times.
+It does not create shards that are either wholly or partially in the past.
```

Comment:
Earlier, we discuss the **shard group duration** determining the interval that new shards/shard groups are created. Can we reword to align with this language? For example, "_InfluxDB pre-creates shards with future start and end times for each shard group duration_" ? 

Alternatively, when discussing shards above, we could clarify the concept of shards being "wholly or partially in the past," but think that may be less intuitive?